tests/test_parallel.py

```diff
@@ -0,0 +1,240 @@
+import copy
+from typing import List
+
+import numpy as np
+import pytest
+
+try:
+    from spyrmsd.parallel import prmsdwrapper
+except ImportError:
+    pytest.skip("Pebble not available", allow_module_level=True)
+
+
+# Results obtained with MDAnalysis
+#   trp0 = mda.Universe("trp0.pdb")
+#   c0 = trp0.coord -trp0.select_atoms("protein").center_of_geometry()
+#   for i in [1,2,3,4,5]:
+#       trp = mda.Universe(f"trp{i}.pdb")
+#       rmsd_dummy = mda.analysis.rms.rmsd(trp.coord, trp0.coord)
+#       c = trp.coord - trp.select_atoms("protein").center_of_geometry()
+#       _, rmsd_min = align.rotation_matrix(tc, tc0)
+#       print(rmsd_dummy, rmsd_min)
+@pytest.mark.parametrize(
+    "minimize, referenceRMSDs",
+    [
+        (
+            False,  # No minimize: dummy RMSD
+            [
+                4.812480551076202,
+                6.772045449820714,
+                9.344911262612964,
+                9.772939589989000,
+                8.901837608843241,
+            ],
+        ),
+        (
+            True,  # Minimize: QCP
+            [
+                1.6578281551053196,
+                1.7175638492348284,
+                1.5946081072641485,
+                2.1234944939308220,
+                2.4894805175766606,
+            ],
+        ),
+    ],
+    ids=["no_minimize", "minimize"],
+)
+def test_prmsdwrapper_nosymm_protein(trps, minimize: bool, referenceRMSDs: List[float]):
+    mol0 = copy.deepcopy(trps[0])
+    mols = [copy.deepcopy(mol) for mol in trps[1:]]
+
+    RMSDs = prmsdwrapper(mol0, mols, symmetry=False, minimize=minimize, strip=False)
+
+    for RMSD, referenceRMSD in zip(RMSDs, referenceRMSDs):
+        assert RMSD == pytest.approx(referenceRMSD)
```

Comment:
I think using [`numpy.testing.assert_allclose`](https://numpy.org/doc/stable/reference/generated/numpy.testing.assert_allclose.html) would be nicer here.

Reply:
That would also check that the sequences have the same lengths, which I think is important since we are manually appending `np.nan`s sometimes.